### PR TITLE
Update macos.md

### PR DESCRIPTION
**Pull request by Kunderscore**
with a very basic example

### CODE REVIEW

File: _docs/examples/macos.md [hunk 1]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
 You may also need to disable [System Integrity Protection](https://support.apple.com/en-us/HT204899).
 
 _Please click "Improve this page" above and add an example. Thanks!_
```
Comment (oleavr) [inline _docs/examples/macos.md]: Let's drop this line and the blank line before it.

File: _docs/examples/macos.md [hunk 2]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
...
+import sys
+
+def on_message(message, data):
+    print("[%s] => %s" % (message, data))
+
+def main(target_process):
+    session = frida.attach(target_process)
+
+    script = session.create_script("""
+	// Get a reference to the openURL selector
```
Comment (oleavr) [inline _docs/examples/macos.md]: - This is `Get a reference to the openURL selector` => `Get a reference to the openURL method`.
- Indentation seems off from the next line.

File: _docs/examples/macos.md [hunk 3]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
...
+  var appWillFinishLaunch = ObjC.classes.NSApplicationDelegate["- applicationWillFinishLaunching:"];
+
+  // Intercept the method
+  Interceptor.attach(appWillFinishLaunch.implementation, {
+    onEnter: function(args) {
+      // As this is an ObjectiveC method, the arguments are as follows:
```
Comment (oleavr) [inline _docs/examples/macos.md]: Nit: `Objective-C`

File: _docs/examples/macos.md [hunk 4]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
...
+      // 0. 'self'
+      // 1. The selector (applicationWillFinishLaunching:)
+      // 2. The first argument to the this selector
```
Comment (oleavr) [inline _docs/examples/macos.md]: `The first argument to this method`

File: _docs/examples/macos.md [hunk 5]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
...
+      var myNotification = new ObjC.Object(args[2]);
+
+      // Convert it to a JS string
+      var myJSNotification = myNotification.absoluteString().toString();
+      // Log it
```
Comment (oleavr) [inline _docs/examples/macos.md]: This comment seems redundant, so perhaps it's more readable to merge the two comments into: `Convert it to a JS string and log it`.

File: _docs/examples/macos.md [hunk 6]
```diff
@@ -11,3 +11,44 @@ If you run your Frida tool via the GUI with your local user (e.g. from Terminal.
 You may also need to disable [System Integrity Protection](https://support.apple.com/en-us/HT204899).
 
 _Please click "Improve this page" above and add an example. Thanks!_
+
+{% highlight py %}
+
```
Comment (oleavr) [inline _docs/examples/macos.md]: Redundant blank line.

**Comment by Kunderscore** [comment]
Anything else ?

**Comment by oleavr** [comment]
Tweaked version merged as 98c0c284d75a05106f961ed94471660233c4400d. Thanks!

**Pull request closed without merging**